packages/pagination/src/mongo-pagination-param.decorator.ts

```diff
@@ -14,8 +14,8 @@ export interface MongoPagination {
   };
   limit: number;
   skip: number;
-  sort: [];
-  project: [];
+  sort: {};
+  project: {};
```

Comment:
Don't you think we can be more specific, like:

```suggestion
  sort?: {
    [key: string]: SortValue;
  } | string;
  project?: {
    [key: string]: 0 | 1;
  };
}
/**
 * Sort values available for Mongoose
 * Ref: https://mongoosejs.com/docs/api/query.html#query_Query-sort
 */
type SortValue = 'asc' | 'desc' | 'ascending' | 'descending' | 1 | -1;
```

Reply:
Good idea 😉  I will change it.